ios/APIClient.swift

```diff
@@ -10,16 +10,21 @@
 import Alamofire
 import SwiftyJSON
 
+let EXPONENTIAL_RETRY = "exponential"
```

Comment:
what's the use of this constant? I saw it in the example file, can this be constant be exposed by the module ?